### PR TITLE
Fix a stupid bug in build stability logic.

### LINKED ISSUE #112: checkBuilds() returns true even when builds are failing?
https://github.com/kubernetes/contrib/blob/023e2c05df5899bf9734b0d6d21fad0b0617fb45/submit-queue/e2e.go#L93

As far as I can tell by looking at this code, this function returns true even if stable==false for some or all builds.

In theory this will cause PR's to be merged even if Jenkins e2e tests are failing.

@gmarek has reported this to be the case and @lavalamp has therefore disabled the merge bot.  

Need to verify that this theoretical bug is in fact manifesting.  The fix is pretty straightforward.

@brendandburns FYI.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes #112
I promise a unit test next, but I have a meeting...

### CODE REVIEW

File: submit-queue/e2e.go [hunk 1]
```diff
@@ -104,6 +104,7 @@ func (e *e2eTester) checkBuilds() (allStable bool) {
 			e.setBuildStatus(build, "Stable")
 		} else {
 			e.setBuildStatus(build, "Not Stable")
+                        allStable = false
```
Comment (lavalamp) [inline submit-queue/e2e.go]: needs a gofmt? (extra space in front)

**Comment by brendandburns** [comment]
ok, gofmt'd and unit test added.  ptal.

File: submit-queue/e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
+package main
```
Comment (lavalamp) [inline submit-queue/e2e_test.go]: Needs the boilerplate. Time to set up the boilerplate checker, sigh...

**Comment by lavalamp** [comment]
LGTM if you fix the missing boilerplate

**Comment by gmarek** [comment]
I don't think the boilerplate was added. @eparis can you please create a PR that does this?

**Comment by eparis** [comment]
https://github.com/kubernetes/contrib/pull/115

**Pull request merged**